README.md

```diff
@@ -9,6 +9,8 @@ Needs following changes:
 SOONG_CONFIG_NAMESPACES += qti_kernel_headers
 SOONG_CONFIG_qti_kernel_headers := version
 SOONG_CONFIG_qti_kernel_headers_version := 4.14
+or
```

Comment:
```suggestion
# or
```

Reply:
But maybe we should only suggest the new helper-based syntax?

Reply:
Because the macro was added not so long ago (A13) I decided to keep both variants, as the current branch is used for earlier versions of Android too.
I will add '#' to the readme a bit later. Have a power outage now🥲

Reply:
Thanks for confirming, wasn't sure which Android versions were still referring to this repository on the `master` branch.